index.js

```diff
@@ -298,14 +748,32 @@ function createSession(user) {
  */
 function findWebClient(circuit) {
   return circuit.getDevices().then(devices => {
-      return devices.find(device => {
-          return (device.clientId !== circuit.user.clientId) &&
-              ((device.clientInfo.deviceType === 'WEB') ||
-              (device.clientInfo.deviceType === 'APPLICATION' && device.clientInfo.deviceSubtype === 'DESKTOP_APP'));
-      });
+    return devices.find(device => {
+      return device.clientId !== circuit.user.clientId && (device.clientInfo.deviceType === 'WEB' || (device.clientInfo.deviceType === 'APPLICATION' && device.clientInfo.deviceSubtype === 'DESKTOP_APP'));
```

Comment:
If you're going to put this into one line you can do:
return devices.find(device => device.clientId !== ... 'DESKTOP_APP'));

Reply:
This change was done automatically by my text editor. The change was reverted.